src/main/kotlin/frc/robot/commandGroups/CommandGroups.kt

```diff
@@ -82,4 +80,43 @@ object CommandGroups {
             (gripper.setRollerPower(-0.7))
         ).withName("outtake")
     }
+
+    fun superPoopInit(): Runnable {
+        return Runnable {
+            Commands.parallel(
+                warmup(
+                    CommandGroupsConstants.HOOD_ANGLE_SUPER_POOP,
+                    CommandGroupsConstants.SHOOTER_VELOCITY_SUPER_POOP,
+                    CommandGroupsConstants.CONVEYOR_VELOCITY_SUPER_POOP
+                ), // TODO: ledMode,
+                swerveDrive.turnCommand(
+                    Units.Degrees.of(
+                        swerveDrive.estimator.estimatedPosition.translation.getRotationToTranslation(
+                            CommandGroupsConstants.SUPER_POOP_TRANSLATION
+                        ).degrees
+                    ),
+                    CommandGroupsConstants.SUPER_POOP_TURN_TOLERANCE.`in`(Units.Rotations)
+                )
+            ).until { readyToSuperPoop() } // TODO: add LEDS
+        }
+    }
+
+    fun superPoopEnd(): Runnable {
+        return Runnable {
+            stopWarmup()
+        }
+        // TODO: LED MODE Default
+    }
+
+    fun superPoop(): Command {
+        return StartEndCommand(
+            superPoopInit(),
+            superPoopEnd(),
+            shooter, hood, conveyor, swerveDrive
+        )
+    }
+
+    fun readyToSuperPoop(): Boolean {
+        return shooter.atSetpoint() && conveyor.atSetPoint() && hood.atSetpoint()
+    }
```

Comment:
The implementation looks very generic, something that is relevant also for non-💩 modes.

Reply:
c734ba28ddcb741841aa695af42dc6efddf43244